examples/todo/src/task.rs

```diff
@@ -1,4 +1,5 @@
-use diesel::{self, prelude::*};
+use diesel::{self, result::Error as DieselError, prelude::*};
+type Result<T> = std::result::Result<T, DieselError>;
```

Comment:
Isn't this `diesel::result::QueryResult`? We should import and return that from these functions instead of duplicating it here.